roxctl/common/flags/endpoint.go

```diff
@@ -114,3 +114,22 @@ func SkipTLSValidation() *bool {
 func CAFile() string {
 	return caCertFile
 }
+
+// CentralURL returns the URL for the central instance based on the endpoint flags.
+func CentralURL() (*url.URL, error) {
+	endpoint, plaintext, err := EndpointAndPlaintextSetting()
```

Comment:
Does the above function cover the case where the server name is passed using the appropriate flag ?